frontend/components/login/index.js

```diff
@@ -178,6 +178,7 @@ class Login extends Component {
                         />
                         <i
                             className="form-text__icon form-text__icon--eye form-text__icon--eye_open"
+                            id="showpasswordbutton"
```

Comment:
All style of naming should be the same. Use kebab-case, so it would be "show-password-button"

Reply:
Fixed